apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/addsection/model/CreateSectionRequest.java

```diff
@@ -0,0 +1,5 @@
+package gov.cdc.nbs.questionbank.addsection.model;
+
+public record CreateSectionRequest(long wa_ui_metadata_uid, long page, String name, String visible) {
```

Comment:
the generic `wa_ui_metadata_uid` field could be changed to something like `tabId` to make it a bit clearer on what type of component it is referencing